rust/feature-flags/src/api/types.rs

```diff
@@ -261,4 +262,88 @@ mod tests {
             expected_description
         );
     }
+
+    #[test]
+    fn test_flags_response_only_includes_non_null_payloads() {
+        // Create a response with multiple flags, some with payloads and some without
+        let mut flags = HashMap::new();
+
+        // Flag with payload
+        flags.insert(
+            "flag1".to_string(),
```

Comment:
The most minor of nits, but I find naming the flags after what we expect of them helps in reading the test. For example, `flag_with_payload`, `flag_without_payload`, `flag_with_null_payload`.

Reply:
I like this idea, will adopt